### PR TITLE
[Chore](Sonar)Set C++ code detection to scheduled scheduling

### PR DESCRIPTION
**Pull request by CalvinKirs**
## Proposed changes
The fork warehouse cannot obtain the token and therefore cannot run.
Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by CalvinKirs** [comment]
run buildall

File: .github/workflows/sonarcloud.yml [hunk 1]
```diff
@@ -85,3 +85,58 @@ jobs:
...
+          sudo DEBIAN_FRONTEND=noninteractive apt install --yes tzdata byacc
+
+          pushd thirdparty
+          curl -L https://github.com/apache/doris-thirdparty/releases/download/automation/doris-thirdparty-prebuilt-linux-x86_64.tar.xz \
+            -o doris-thirdparty-prebuilt-linux-x86_64.tar.xz
+          tar -xvf doris-thirdparty-prebuilt-linux-x86_64.tar.xz
+          popd
+
+          export PATH="${DEFAULT_DIR}/ldb-toolchain/bin/:$(pwd)/thirdparty/installed/bin/:${PATH}"
+          DISABLE_JAVA_UDF=ON DORIS_TOOLCHAIN=clang OUTPUT_BE_BINARY=0 ./build.sh --be
```
Comment (BiteTheDDDDt) [inline .github/workflows/sonarcloud.yml:128]: add ENABLE_PCH=OFF on here

**Review by BiteTheDDDDt — APPROVED** [review]
LGTM

**Comment by CalvinKirs** [comment]
run buildall

**Review by zclllyybb — APPROVED** [review]

**Pull request merged**